spectrum/autism_prevalence_map/templates/autism_prevalence_map/action_buttons.html

```diff
@@ -37,6 +37,24 @@
                     <option value="high">&gt;100,000</option>
                 </select>
             </div>
+            <div class="form-group">
+                <label class="" for="meanincomeofparticipants">Income</label>
+                <select class="custom-select" id="meanincomeofparticipants">
+                    <option value="all">All</option>
+                    <option value="Low">Low</option>
+                    <option value="Medium">Medium</option>
+                    <option value="High">High</option>
```

Comment:
it's weird to me that these values are capitalized. is that intentional?

Reply:
It is both intentional and weird. Just because that's how they are in the spreadsheet at the moment. I would like to clean it up some and make it more case-agnostic on the Django end, but am still getting familiar with how it's all set up